components/base/sensorcontrolled/sensorcontrolled.go

```diff
@@ -227,6 +227,10 @@ func (sb *sensorBase) Stop(ctx context.Context, extra map[string]interface{}) er
 	sb.opMgr.CancelRunning(ctx)
 	if sb.loop != nil {
 		sb.loop.Pause()
+		// update pid controllers to be an at rest state
+		if err := sb.updateControlConfig(ctx, 0, 0); err != nil {
+			return err
+		}
```

Comment:
kinda forgot that as a user I could just tell my controller to stop. so im gonna do that now. 

running into a small issue where if I have a sensorcontrolled base and controlled motor running, calling stop will break the motor, but I can probably track that down

Reply:
is this issue resolved?

Reply:
yep the issue is fixed now, the reason it was happening was because I set a velocity of 0. I didn't notice that at first because Next wasn't being locked, so reset would occur during next calls